how-to-delete-browser-cookies/script.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * The list of cookies to keep
+ */
+(function () {
+  var itemsToKeep = ["euconsent-v2", "didomi_token"];
+  var itemsToKeepRegex = /some_regex_[a-z0-9]*/;
```

Comment:
`itemsToKeepRegex` is just example if so maybe need to mention about it in README.md We give user the ability to keep other tokens apart of  `["euconsent-v2", "didomi_token"]` right ?